src/sql_generator.cpp

```diff
@@ -122,9 +125,10 @@ std::vector<column_def> describe_table(duckdb::Connection &con,
   // TBD scale/precision
   std::vector<column_def> columns;
 
-  auto query = "SELECT column_name, data_type, is_nullable = 'NO' FROM "
-               "information_schema.columns WHERE table_catalog=? AND "
-               "table_schema=? AND table_name=?";
+  auto query = "SELECT column_name, data_type_id, NOT is_nullable, "
+               "numeric_precision, numeric_scale FROM "
+               "duckdb_columns() WHERE database_name=? AND "
+               "schema_name=? AND table_name=?";
```

Comment:
minor nit, I feel this would be more readable: (and future diffs too)
```suggestion
  auto query = "SELECT "
               "  column_name,"
               "  data_type_id,"
               "  NOT is_nullable, "
               "  numeric_precision,
               "  numeric_scale "
               "FROM duckdb_columns()"
               "WHERE database_name=? "
               "  AND schema_name=?" 
               "  AND table_name=?";
```

Reply:
done